### DISCUSSION TITLE
Output .avif file seems washed out

### DISCUSSION THREAD

**Discussion by everthis** [ImageMagick/ImageMagick#7302] [Help]
### ImageMagick version

7.1.1-32

### Operating system, version and so on

OS: Ubuntu 22.04.3 LTS x86_64

### Description

```
$ magick -version
Version: ImageMagick 7.1.1-32 Q16-HDRI x86_64 22207 https://imagemagick.org
Copyright: (C) 1999 ImageMagick Studio LLC
License: https://imagemagick.org/script/license.php
Features: Cipher DPC HDRI OpenMP(4.5)
Delegates (built-in): fontconfig freetype heic png x xml zlib
Compiler: gcc (11.4)
```

```
magick hdr.avif -resize 50% out.avif
```
The `hdr.avif` is from 
[https://raw.githubusercontent.com/AOMediaCodec/av1-avif/master/testFiles/Netflix/avif/hdr_cosmos01650_cicp9-16-9_yuv444_full_qp10.avif](https://raw.githubusercontent.com/AOMediaCodec/av1-avif/master/testFiles/Netflix/avif/hdr_cosmos01650_cicp9-16-9_yuv444_full_qp10.avif)

The screenshot below is comparison of original `hdr.avif`(left) and `out.avif`(right).
It's obvious that the output file is not expected.
Am I missing something?

![Screenshot 2024-05-09 231051](https://github.com/ImageMagick/ImageMagick/assets/3142886/433825c9-f9d0-4a69-900a-6e5e0d7a0ba1)

**Comment by fmw42** [comment]
I get the same result on IM 7.1.1.32 on Mac OSX Ventura.

**Comment by snibgo** [comment]
> The screenshot below ...

What software did you use to read the file and display the image?

When I use Gimp, the result looks like the "washed-out" image.

**Reply by everthis** [reply]
Chrome, which support avif files.

**Comment by fmw42** [comment]
How was the image captured?  Was there any processing done?  If from a camera, was the information about auto-brightness lost.  I do not see any camera information in the verbose information.

**Reply by everthis** [reply]
It was captured using windows screen capture tool.And HDR mode in Windows is turned on.

**Reply by fmw42** [reply]
Sounds like you may have lost information about the original when you did a screen capture.  Can you send the original before the screen capture?

**Reply by everthis** [reply]
https://raw.githubusercontent.com/AOMediaCodec/av1-avif/master/testFiles/Netflix/avif/hdr_cosmos01650_cicp9-16-9_yuv444_full_qp10.avif

**Reply by everthis** [reply]
And I think it’s not about screenshot. I took the screenshot because the output is abnormal. The screenshot is just a way to show the difference.

**Comment by everthis** [comment]
Tried with `MacOS 14.2.1 (23C71)`

```
Version: ImageMagick 7.1.1-32 Q16-HDRI aarch64 22207 https://imagemagick.org
Copyright: (C) 1999 ImageMagick Studio LLC
License: https://imagemagick.org/script/license.php
Features: Cipher DPC HDRI Modules OpenMP(5.0) 
Delegates (built-in): bzlib fontconfig freetype gslib heic jng jp2 jpeg jxl lcms lqr ltdl lzma openexr png ps raw tiff webp xml zlib zstd
Compiler: gcc (4.2)
```

Command:
```
magick hdr.avif -resize 50% out.avif
```

The same result:
![20240510-110905](https://github.com/ImageMagick/ImageMagick/assets/3142886/d183aa4d-d9c0-4d89-8f0b-1a59552f8bc0)

**Comment by snibgo** [comment]
I suppose the input image contains HDR information, perhaps in the form of a base SDR image, and a gain map to transform the base image to HDR.

The given command ...
```
magick hdr.avif -resize 50% out.avif
```
... reads all images from hdr.avif, then resizes those images by 50%, then writes those images to out.avif. I suppose it is reading only the SDR image from the input, and that is the only image that is written to the output.

Gimp is doing the same, reading only the SDR image.

I don't know if this is a limitation of libheif or of IM and Gimp.

**Reply by kmilos** [reply]
There is no gain map here. [This is simply an issue of IM not supporting CICP color profiles (i.e. `nclx` box), so the converted out.avif gets tagged as sRGB by default, while the input was Rec. 2100 PQ](https://github.com/ImageMagick/ImageMagick/issues/7308#issuecomment-2112016430).
